OAuth2Client/LocalAuthorizationServer.py

```diff
@@ -32,6 +32,11 @@ def start(self, verification_code: "str") -> None:
         Starts the local web server to handle the authorization callback.
         :param verification_code: The verification code part of the OAuth2 client identification.
         """
+        if self._web_server:
+            # If the server is already running (because of a previously aborted auth flow), we don't have to start it.
```

Comment:
I think this code should be there anyway, but shouldn't we also try to stop the server when the flow is aborted?

Reply:
We don't know when the flow is aborted. With aborted I mean the callback was never reached. A "deny" will already stop the server.